packages/autorest.python/samples/specification/azure_key_credential/generated/azure/key/credential/sample/aio/operations/_operations.py

```diff
@@ -107,15 +107,15 @@ async def head204(self, **kwargs: Any) -> None:  # pylint: disable=inconsistent-
         _headers = kwargs.pop("headers", {}) or {}
         _params = kwargs.pop("params", {}) or {}
 
-        cls = kwargs.pop("cls", None)  # type: ClsType[None]
+        cls: ClsType[None] = kwargs.pop("cls", None)
 
         request = build_http_success_head204_request(
             headers=_headers,
             params=_params,
         )
-        request.url = self._client.format_url(request.url)  # type: ignore
+        request.url = self._client.format_url(request.url)
 
-        pipeline_response = await self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
+        pipeline_response: PipelineResponse = await self._client._pipeline.run(  # pylint: disable=protected-access
```

Comment:
I think we have other code that toggles between commented out types and inline types, can you make sure we don't allow commented types anymore?

Reply:
Pyright report `error: Use of type comments is deprecated; use type annotation instead (reportTypeCommentUsage)` on commented types. So, I removed commented types as much as it's reported by current pyright checking. 

I can find document for this error at here: https://github.com/microsoft/pyright/blob/main/docs/configuration.md. I think it's safe for us not to use comment types since our SDK are now for python 3.6+.
Alternatively, we can disable this error at https://github.com/Azure/autorest.python/blob/20221019-pyright/packages/autorest.python/pyrightconfig.json#L4 if we want to keep commented types.

Reply:
We should aim to get rid of all type comments

Reply:
Hi @iscai-msft ,  I cleaned up all commented types in latest two commits :)